python/ccf/cose.py

```diff
@@ -124,20 +127,78 @@ def create_cose_sign1(
     return msg.encode()
 
 
-DESCRIPTION = """Create and sign a COSE Sign1 message for CCF governance
+def create_cose_sign1_prepare(
+    payload: bytes,
+    cert_pem: Pem,
+    additional_headers: Optional[dict] = None,
```

Comment:
Each item is called a header parameter. The header is the bucket (dict) itself.

Reply:
I see, changing to `additional_protected_header`, for clarity.